### ISSUE TITLE
pre-trained model for c3d

### ISSUE THREAD

**Issue by hekim3434** [facebook/C3D#281]
Is there any pre-trained model for C3D-v1.1 trained with Sport1M or UCF 101 dataset?
(I downloaded a pre-trained c3d resnet model trained with UCF 101 from this repository)
Also, when training c3d model from scratch, how long should I train the model to have top-1 accuracy over 80%?

For example,  top-1 accuracy of the c3d model in the  "examples/c3d_ucf101_training", which I trained from scratch for 60,000 iteration was 33.17% (it was tested on the provided list of validation set with the batch size 1).
However, in the C3D UserGuide (from http://vlg.cs.dartmouth.edu/c3d/ ), the clip accuracy of the fine-tuned model with pre-trained is 80.19%. 

when training c3d from scratch, would it be possible to have similar accuracy of the fine-tuned model if I train the model with longer iteration?

**Comment by dutran** [comment]
C3D v1.0 (VGG architecture) can be found here: http://vlg.cs.dartmouth.edu/c3d/ both for Sports1M and fine-tuned on UCF101.

for C3D v1.1 (Resnet18 architecture) can be found here for Sports1M (https://github.com/facebook/C3D/blob/master/C3D-v1.1/examples/c3d_ucf101_feature_extraction/feature_extraction.sh#L2) or fine-tune on UCF101 (https://www.dropbox.com/s/bf5z2jw1pg07c9n/c3d_resnet18_ucf101_r2_ft_iter_20000.caffemodel?dl=0)
